### ISSUE TITLE
Add multiple language on tab

### ISSUE THREAD

**Issue by ChrisOlv** [sumpfork/dominiontabs#531]
Hi Dominion lovers, 

Do you think it would be possible to add an option to enable orginal title next to the title ? "title_FR (original_title)"

Exemple : 
Aventurier (Adventurer)
Recrue (Hireling)

I play with french cards but I often play with kingdoms found on reddit or on podcast and It's a pain to translate everything.

Thank you !

**Comment by sumpfork** [comment]
This would be a bit of an undertaking because we don't load the English text if another language is being generated, and the layout and spacing would probably be affected.

So, not impossible, but unlikely unless someone else has time to address this request.

**Comment by ChrisOlv** [comment]
Ok, I will try to play around with the source code.

**Comment by mbbush** [comment]
> we don't load the English text if another language is being generated

That's true, but we do have the English card titles as the card tag. I've already got a change locally that standardizes those to match the English titles, maybe that could be used fairly easily to power this option? 

If I get a chance, I may take a stab at this.
